lib/page-object/platforms/watir_webdriver/page_object.rb

```diff
@@ -970,6 +986,20 @@ def find_watir_element(the_call, type, identifier, tag_name=nil)
           type.new(element, :platform => :watir_webdriver)
         end
 
+        def find_watir_pages(identifier, page_class)
```

Comment:
probably has a lot of duplication with the method for finding an element. It is likely worth extracting a private method, but I didn't know if you would want me to take it that far.